Makefile

```diff
@@ -0,0 +1,39 @@
+ # Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+DOCKER=docker
+BINARY_NAME=adapter
+TARGET=target
+BINARY_LINUX=$(TARGET)/$(BINARY_NAME)_linux
+BINARY_DARWIN=$(TARGET)/$(BINARY_NAME)_darwin
+BINARY_WINDOWS=$(TARGET)/$(BINARY_NAME)_windows.EXE
+
+all: build test
+build: 
```

Comment:
Perhaps `build:deps` ?

Reply:
In my opinion, you want ```make build``` to be as fast as possible when you're in a edit-build-debug cycle, so it's often considered best practice to keep deps out of the build target.

HOWEVER, the "all" target should definitely have a reference to deps. Will fix that.

Reply:
Fixed